EmployeesSalaries/EmployeesSalaries/Models/Employee/HR.cs

```diff
@@ -0,0 +1,30 @@
+﻿using EmployeesSalaries.Models.Interfaces;
+
+namespace EmployeesSalaries.Models.Employee
+{
+    public class HR : IEmployee, IReportsTo
+    {
+        public string FirstName { get; set; } = "Test";
+        public string LastName { get; set; } = "Test";
+        public IEmployee Supervisor { get; set; } = new Manager();
+
+        public int Id { get; } = 3;
```

Comment:
create new class that contains all the data, it will be replaced later on with database

Reply:
Data moved to a separate class